src/explore-education-statistics-common/src/modules/methodology/components/__tests__/MethodologyHelpAndSupportSection.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import * as ContactUsSection from '@common/modules/find-statistics/components/ContactUsSection';
+import MethodologyHelpAndSupportSection from '../MethodologyHelpAndSupportSection';
+import {
+  mockMethodologyPublication,
+  mockPublicationSummary,
+} from './__data__/mock-data';
+
+jest.mock('@common/modules/find-statistics/components/ContactUsSection');
+
+describe('Methodology Help and Support Section (PublicationSummary)', () => {
+  beforeEach(() => {
+    jest
+      .spyOn(ContactUsSection, 'default')
+      .mockReturnValue(<div>This is a mocked ContactUsSection</div>);
+
+    render(
+      <MethodologyHelpAndSupportSection
+        owningPublication={mockPublicationSummary}
+      />,
+    );
+  });
+
+  it('Renders', () => {
+    expect(
+      screen.getByRole('heading', { name: /Help and support/ }),
+    ).toBeVisible();
+  });
+
+  it('Renders a Contact Us Section', () => {
+    expect(screen.getByText('This is a mocked ContactUsSection')).toBeVisible();
+  });
+});
+
+describe('Methodology Help and Support Section (mockMethodologyPublication)', () => {
```

Comment:
Happy to discuss this in person sometime, as it is something I went back and forth on during development. 

Ultimately what sways me towards doing it this way is clarity for future devs. The signature `<MethodologyHelpAndSupportSection publicationContact={publication.contact} publicationTitle={publication.title} />` is a) overly verbose and b) doesn't communicate very well that the contact and title are a tightly bound concept. In fact I originally passed in the methodology title until I became more familiar with the codebase. 

I think `<MethodologyHelpAndSupportSection owningPublication={publication}/>` is a) more readable and b) clearly communicates that it's the owning publication this component requires/cares about. 

I do see both sides though, so as I say happy to discuss, but this was my reasoning.

Reply:
I'm okay with the current approach now that we only take a single type of `owningPublication` and don't have a duplicated set of tests. When we have these kind of situations, it's a usually a code smell that I'd resolve with the type system instead of tests.